pai-management/doc/how-to-write-pai-configuration.md

```diff
@@ -222,9 +224,9 @@ hadoop:
       capacity: 20
 ```
 
-- ```custom-hadoop-binary-path```: If you wanna to use hadoop-ai (A hadoop patched version developed by PAI. [More detail](../../hadoop-ai)) please set a path here. And paictl will build hadoop-ai, and place the binary to the path. If you don't wanna use hadoop-ai, set ```None``` here.
-- ```hadoop-version```: If you will build hadoop-ai and use it, you should set this as ```2.7.2```. If you set path as ```None```, please choose a version [here](http://archive.apache.org/dist/hadoop/common/).
-- ```virtualClusters```: hadoop queue setting.
+- ```custom-hadoop-binary-path```: If you would like to use [hadoop-ai](../../hadoop-ai), please set a path here. And paictl will build hadoop-ai, and place the binary to the path. If you do not use hadoop-ai, set ```None``` here.
+- ```hadoop-version```: If you will build hadoop-ai and use it, you should set this to ```2.7.2```. If you set path as ```None```, please choose a version [here](http://archive.apache.org/dist/hadoop/common/).
+- ```virtualClusters```: hadoop queue setting. The sum of capacity should be 100.
```

Comment:
Not necessary. The logic is as follows:

- Each VC will be assigned with (capacity / total_capacity * 100%) of the resources in the system.
- The system will automatically create the 'default' VC with 0 capacity, if 'default' VC has not been explicitly specified in the configuration file.
- The system will split resources to each VC evenly if the total capacity is 0.
- The capacity of each VC will be automatically set to 0 if it is a negative number in the configuration file.

Please see: https://github.com/Microsoft/pai/pull/689